image/templates/CHANGING_CHARTS.md

```diff
@@ -1,5 +1,44 @@
 # Changing StackRox Helm charts
 
+## Helm Template Coding Style Guidelines
+
+### Whitespace Elimination
+
+Given that the sole purpose of the Helm templating is to render out Kubernetes manifests in YAML format,
+it is critical that the rendering process does not mangle with whitespace indentation in an undesired manner.
+
+Helm templating -- just like the underlying Go templating engine -- supports different mechanisms for
+whitespace trimming, see [Go templating docs](https://pkg.go.dev/text/template#hdr-Text_and_spaces) for more information on this.
+
+For our Helm charts we have settled on the following coding style:
+
+For in-line interpolation we usually do not require any whitespace trimming. For example, we can write the templating code as follows:
```

Comment:
nit: "we usually do not require" - are there exceptions?

Reply:
I didn't want to formulate it too strict, because, well, for readability purposes one could e.g. use something like this:

```
foo: {{ if ... -}} 1 {{- else -}} 2 {{- end }}
```

instead of:
```
foo: {{ if ... }}1{{ else }}2{{ end }}
```

Reply:
@vlad ok to merge?